cypress/integration/executions_spec.js

```diff
@@ -414,6 +414,7 @@ describe('Dashboard Executions Page', () => {
       cy.get('@executionGraphNodes').eq(1).should('have.text', 'HelloWorld');
 
       cy.contains('.sidebar__nav--back', 'Back to Executions').click();
+      cy.wait(1000);
```

Comment:
Why did we need a delay?

Reply:
Waiting for an element to render appears to be a common pattern throughout the tests - this one makes less sense than the others, but I was seeing intermittent failures on CI / locally so I'm assuming it was an existing issue and solved it in-line with the other code in the repo.